cdep/src/main/java/io/cdep/cdep/generator/CMakeGenerator.java

```diff
@@ -62,6 +64,13 @@
   final private FunctionTableExpression table;
   @NotNull
   private StringBuilder sb;
+  private ArrayList<String> dependsList;
```

Comment:
Are these new fields strongly coupled enough to merit a named nested class to hold them?

Reply:
Not a bad thought; should be easier to read too.  I'll add one.